### PR TITLE
Support Ledger Live legacy Ethereum derivation

### PR DESCRIPTION
**Pull request by marnova**

### CODE REVIEW

File: core/src/apps/common/paths.py [hunk 1]
```diff
@@ -334,6 +334,9 @@ def match(path: Bip32Path) -> bool:
 PATTERN_BIP44 = "m/44'/coin_type'/account'/change/address_index"
 # BIP-44 public key export, starting at end of the hardened part
 PATTERN_BIP44_PUBKEY = "m/44'/coin_type'/account'/*"
+# BIP-44 Ledger Live legacy path
+# https://github.com/trezor/trezor-firmware/issues/1749
+PATTERN_BIP44_LEDGER_LIVE_LEGACY = "m/44'/coin_type'/0'/account"
```
Comment (mmilata) [inline core/src/apps/common/paths.py:339]: Let's maybe rename this to `PATTERN_SEP5_LEDGER_LIVE_LEGACY` since ethereum is not UTXO based?

Comment (marnova) [reply core/src/apps/common/paths.py]: 2d095f485f3c81523e2794d4ad1c149897fb91c0

**Review by mmilata — APPROVED** [review]
Looks good to me!

**Review by matejcik — APPROVED** [review]
@marnova please squash the fixup and merge

**Pull request merged**